### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by wendigo**
Housekeeping :)

### CODE REVIEW

**Comment by wendigo** [comment]
RoaringBitmap reached 1.0.0 (https://github.com/RoaringBitmap/RoaringBitmap/releases/tag/1.0.0)

**Comment by hashhar** [comment]
please consider automation.

**Comment by wendigo** [comment]
@hashhar I did ask but there were no agreement that we want to do that

**Comment by wendigo** [comment]
@jklamer for avro update :)

**Comment by wendigo** [comment]
Updated to airbase 146.

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>144</version>
+        <version>146</version>
```
Comment (electrum) [inline pom.xml:8]: We can remove the override for `dep.junit` now

Comment (electrum) [reply pom.xml:8]: Also remove `dep.jhm`

**Review by electrum — APPROVED** [review]

**Comment by wendigo** [comment]
I've dropped airbase update as it seems that some of the PT failures (seems like memory leak) are related to it. I'm going to triage that

**Pull request merged**